storybook/config.ts

```diff
@@ -1,7 +1,10 @@
 import { addDecorator, addParameters, configure } from "@storybook/react"
 import { create } from "@storybook/theming"
+import focusVisible from "focus-visible"
 import { backgrounds } from "./backgrounds"
 
+focusVisible // This causes the :focus-visible polyfill to load
```

Comment:
I'm interested if this is actually needed... a line like this _should_ have no side effects